src/ranged.cpp

```diff
@@ -948,43 +943,42 @@ static void do_aim( player *p, std::vector <Creature *> &t, int &target,
         p->recoil = std::max(MIN_RECOIL, p->recoil);
     }
 
-    const int aim_amount = p->aim_per_time( *relevant, p->recoil );
+    const double aim_amount = p->aim_per_move( *relevant, p->recoil );
     if( aim_amount > 0 ) {
         // Increase aim at the cost of moves
-        p->moves -= 10;
+        p->moves--;
```

Comment:
Yes, but when play tested there is no latency. The only reason we had the `10` part was because `recoil` was previously an `int` and lacked sufficient granularity (see the deleted function documentation).